HISTORY.rst

```diff
@@ -4,6 +4,9 @@ History
 
 ##### (##########)
 ------------------
+* Allow the use of `y` and `groups` arguments in cross validator methods `get_n_splits`
+  and `split` to enable more cv-split variants for :class:`MapieRegressor`
+  (e.g. :class:`GroupKFold`, stratified continuous split).
```

Comment:
You can add that you also make this change in the classification file.

Reply:
Fixed by listing methods with signature change.